src/server/schema/resolvers/types/Life.ts

```diff
@@ -0,0 +1,8 @@
+import { GraphQLLifeResolvers } from '../definitions';
+
+const LifeGraphQL: GraphQLLifeResolvers = {
+    fullName: root => `${root.firstName} ${root.lastName}`,
+    id: root => root._id,
```

Comment:
oh you already did it here, no need to do inside the query

Reply:
yeah forgot about this one hahaha.